src/calculator/Calculator.tsx

```diff
@@ -0,0 +1,11 @@
+import styles from './calculator.module.css';
+import textStyles from 'src/components/text/text.module.css';
```

Comment:
instead of importing the textStyles you can use the `<Text />` component as `<Text />` for body and `<Text type="small" />`

not important for this case as it's all just placeholder for now, but later it's important! ⭐